### PR TITLE
feat(route): 新增 和讯创投

### PR DESCRIPTION
**Pull request by p3psi-boo**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/hexun/pe/news
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/hexun/index.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+const decoder = new TextDecoder('gbk');
+
+export const route: Route = {
+    path: '/pe/news',
+    categories: ['new-media'],
+    example: '/hexun/pe/news',
+    url: 'https://www.hexun.com/pe/news',
```
Comment (TonyRL) [inline lib/routes/hexun/index.ts:14]: this url does not work

File: lib/routes/hexun/index.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    const $ = load(decoder.decode(response.data));
+
+    const list = $('.listNews li')
+        .toArray()
+        .map((item) => {
+            const element = $(item);
+            const a = element.find('a');
+
+            const link = a.attr('href') || '';
```
Comment (TonyRL) [inline lib/routes/hexun/index.ts:37]: Please use HTTPS instead. [Log](https://github.com/DIYgod/RSSHub/actions/runs/12250435126/job/34173385833#step:13:71)

File: lib/routes/hexun/index.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            const title = a.text() || '';
+            const dateText = link.split('/')[3];
+            const pubDate = dateText ? parseDate(dateText, 'YYYY-MM-DD') : null;
```
Comment (TonyRL) [inline lib/routes/hexun/index.ts:40]: The site provide an exact time string. Please use that instead.

File: lib/routes/hexun/index.ts [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+const decoder = new TextDecoder('gbk');
+
+export const route: Route = {
+    path: '/pe/news',
+    categories: ['new-media'],
```
Comment (TonyRL) [inline lib/routes/hexun/index.ts:12]:
```suggestion
    categories: ['finance'],
```

File: lib/routes/hexun/index.ts [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+const decoder = new TextDecoder('gbk');
+
+export const route: Route = {
+    path: '/pe/news',
+    categories: ['finance'],
+    example: '/hexun/pe/news',
+    url: 'https://pe.hexun.com/news/',
```
Comment (TonyRL) [inline lib/routes/hexun/index.ts:14]: Do not include protocol in `url`.

Comment (p3psi-boo) [reply lib/routes/hexun/index.ts]: Fixed

**Pull request merged**